### PR TITLE
Additional usage of a PooledHashset.

### PR DESCRIPTION
**Pull request by AdamSpeight2008**

### CODE REVIEW

File: src/Compilers/VisualBasic/Portable/Compilation/SemanticModel.vb [hunk 1]
```diff
@@ -909,7 +909,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     End If
                 Next
 
-                Return ImmutableArray.CreateRange(symbolSet)
+                Dim tmp = ImmutableArray.CreateRange(symbolSet)
```
Comment (AlekseyTs) [inline src/Compilers/VisualBasic/Portable/Compilation/SemanticModel.vb:912]: >tmp [](start = 20, length = 3)

Consider using more descriptive name. For example, "result".

**Comment by AlekseyTs** [comment]
Done with review pass (iteration 1)

File: src/Compilers/VisualBasic/Portable/Compilation/SemanticModel.vb (hunk 1 above)
Comment (AdamSpeight2008) [reply src/Compilers/VisualBasic/Portable/Compilation/SemanticModel.vb]: Done.

**Review by AlekseyTs — APPROVED** [review]
LGTM (iteration 2)

**Comment by AlekseyTs** [comment]
@dotnet/roslyn-compiler Please review, need a second sign-off.

**Review by jcouv — APPROVED** [review]
LGTM Thanks (iteration 2)

**Comment by jcouv** [comment]
Merged/squashed. Thanks @AdamSpeight2008

**Pull request merged**